PacketDelivery-ReferenceExample/i4Trust-Marketplace/values/keycloak/templates/issuerKeyConfigMap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "keycloak.fullname" . }}-key
+data:
+  keyfile.json: |-
+    {
+      "kty": "OKP",
```

Comment:
We should remove the key data here. 
Also placing a comment (e.g., in the README), that either the didConfigMap should be used (when using did:web), or the keyIssuerConfigMap (when using did:key). 
In the latter case, we might add a link to the walt-id API spec (https://docs.walt.id/v/ssikit/getting-started/rest-apis/core-api) for creating DIDs and keys. Such keys might be used in the keyfile.json.